dockerfile

```diff
@@ -46,7 +46,7 @@ RUN /usr/local/venv/bin/pip install \
     "dbt-redshift==1.0.0" \
     "dbt-snowflake==1.0.0" \
     "dbt-bigquery==1.0.0" \
-    "dbt-spark[ODBC]==1.0.0"
```

Comment:
@ZeRego do we not need ODBC at all to power the databricks driver?

Reply:
The ODBC driver is installed separately in line 11 and still used in the warehouse adapter.

```# odbc - databricks
RUN apt-get update && apt-get install -y --no-install-recommends \
    wget \
    libsasl2-modules-gssapi-mit
RUN wget \
    --quiet \
    https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/odbc/2.6.18/SimbaSparkODBC-2.6.18.1030-Debian-64bit.zip \
    -O /tmp/databricks_odbc.zip \
    && unzip /tmp/databricks_odbc.zip -d /tmp \
    && dpkg -i /tmp/simbaspark_*.deb
    ```